modules/installation-aws-limits.adoc

```diff
@@ -16,30 +16,29 @@ to request additional resources for you AWS account.
 The following table summarizes the AWS components whose limits can impact your
 ability to install and run {product-title} clusters.
 
-
 [cols="2a,3a,3a,8a",options="header"]
 |===
 |Component |Number of clusters available by default| Default AWS limit |Description
 
-|Instance Limits
-|Varies
-|Varies
+|vCPU limits
+|0
+|1 vCPU
```

Comment:
Here "Number of clusters available by default" - 0, "Default AWS limit" - 1, but the following description is saying "Each default cluster requires 20 vCPU", and I do not think default aws limit only allow 1 vCPU.